src/AdminSettings.vue

```diff
@@ -175,24 +172,21 @@
 </template>
 
 <script>
-import NcMultiselect from '@nextcloud/vue/dist/Components/NcMultiselect.js'
-import NcSettingsSection from '@nextcloud/vue/dist/Components/NcSettingsSection.js'
-import NcCheckboxRadioSwitch from '@nextcloud/vue/dist/Components/NcCheckboxRadioSwitch.js'
-import NcTextField from '@nextcloud/vue/dist/Components/NcTextField.js'
+import { NcSelect, NcSettingsSection, NcCheckboxRadioSwitch, NcTextField } from '@nextcloud/vue'
```

Comment:
I recommend the opposite to reduce bundle size

Reply:
I thought tree-shaking worked properly nowadays ?

Reply:
🤷🏼 reality says it matters

Reply:
> I thought tree-shaking worked properly nowadays ?

Not well with `@nextcloud/vue`. but the most problematic would be build time, IMO. For exanmple:

For example, with `server`:
x | Bundle Size | Build Time
---|---|---
**Separate import** | 225.08 MB | 28s
**Index import** | 199.33 MB | 71s